apps/api/src/config/env.validators.ts

```diff
@@ -19,8 +19,6 @@ export const envValidators = {
   REDIS_TLS: json({ default: undefined }),
   JWT_SECRET: str(),
   SENDGRID_API_KEY: str({ default: '' }),
-  /** @deprecated - use `MONGO_AUTO_CREATE_INDEXES` instead */
-  AUTO_CREATE_INDEXES: bool({ default: false }),
```

Comment:
@tatarco Can we update here as well instead of removing entirely? same for worker as well.